tox_travis.py

```diff
@@ -67,10 +68,33 @@ def guess_python_env():
     return 'py{major}{minor}'.format(major=major, minor=minor)
 
 
+def get_default_envlist(version):
+    """Parse a default tox env based on the version.
+
+    The version comes from the ``TRAVIS_PYTHON_VERSION`` environment
+    variable. If that isn't set, then use sys.version_info to come
+    up with a reasonable default.
+    """
+    # If version isn't set, then guess from sys.version_info
+    if not version:
+        return guess_python_env()
```

Comment:
Upon a bit further thought, I don't think using it on exceptions will really help, because the `TOXENV` is set by this project, but the error is thrown directly by `tox`. The paths here are pretty reliable.

I'll still add the `nightly` special case, though.